tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -92,6 +92,29 @@ Feature: full text search
       | spaces           |
 
 
+  Scenario Outline: search project space folders by tag
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "tag-space" with the default quota using the GraphApi
+    And user "Alice" has created a folder "spacesFolder/spacesSubFolder" in space "tag-space"
+    And user "Alice" has created a folder "unTagSpacesFolder/unTagSpacesSubFolder" in space "tag-space"
+    And user "Alice" has created the following tags for folder "spacesFolder" of the space "tag-space":
+      | tag1 |
+    And user "Alice" has created the following tags for folder "spacesFolder/spacesSubFolder" of the space "tag-space":
+      | tag1 |
+    And using <dav-path-version> DAV path
+    When user "Alice" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
+      | spacesFolder    |
+      | spacesSubFolder |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
```

Comment:
Seems like this doesn't work for spaces in stable
```feature
    Examples:
      | dav-path-version |
      | old              |
      | new              |
      | spaces           |
        Failed step: And the search result of user "Alice" should contain only these files:
        The number of elements in the response doesn't matches with expected number of elements
        Failed asserting that 0 matches expected 2.
```
```suggestion
```

Reply:
ok I will keep the tag `@skipOnStable3.0`

Reply:
@nabim777  does it not work in stable-3.0 or what? it seems to be passing in master isn't it?

Reply:
Yes I think it works for master. I think it this case we can just remove the `stable` option from the example if it's not supposed to work for stable insted to skipping the whole scenario , but is it supposed to not work for stable? cc @saw-jan

Reply:
There was same [issue in master](https://github.com/owncloud/ocis/issues/6669) also later after adding `remote.php` in spaces dav endpoint that works fine and passes. But in here after adding the `remote.php` it gives `405` status

> @nabim777 does it not work in stable-3.0 or what? it seems to be passing in master isn't it?